### PR TITLE
Enabled cross-building for scala 2.11 & 2.12; updated to use most recent akka

### PR DESCRIPTION
**Pull request by tkroman**
I also bumped the version if it's OK with you.

I also couldn't find a published version and judging by the build.sbt it looks like users are supposed to do it manually. Do you mind publishing the library to bintray? If you don't have time or don't want to, are you OK with me publishing this under a different organization (or maybe you can suggest something else)?

### CODE REVIEW

**Comment by chadselph** [comment]
Yeah, I have been meaning to publish it somewhere, I need to learn how to do it. And also to choose a good organization for myself. I was a bit worried that publishing it might make it seem more stable than it really is. I don't work on this project too often, but when I do I tend to make sweeping changes.

File: build.sbt [hunk 1]
```diff
@@ -2,22 +2,23 @@ name := "Akka SMPP"
...
-version := "0.3.0-SNAPSHOT"
+version := "0.4.0-SNAPSHOT"
 
-scalaVersion := "2.11.8"
+scalaVersion := "2.12.1"
 
-resolvers += "Typesafe Repository" at "http://repo.typesafe.com/typesafe/releases/"
+crossScalaVersions := Seq("2.11.9", "2.12.1")
 
+resolvers += "Typesafe Repository" at "http://repo.typesafe.com/typesafe/releases/"
```
Comment (chadselph) [inline build.sbt]: Is this actually needed anymore? It might be leftover from when I was pulling down RCs or something

Comment (tkroman) [inline build.sbt]: I'm not sure, will try without it.

Regarding publishing - the current namespace (akkasmpp) seems OK. The process is a pain in the ass, though. You'll have to register on sonatype and read like 10 random articles on the internet to get it right, so be ready to spend quite some time on it. I did it recently, so feel free to contact me if you'll feel the need.

Comment (chadselph) [inline build.sbt]: Yea, I think I got about halfway there but for some reason I didn't like the org being called `akkasmpp`. Well... it's the best I got so I'm going with it for now.

**Comment by chadselph** [comment]
ok, pushed to my bintray:

https://bintray.com/chadselph/maven/akka-smpp

apparently it doesn't support snapshots, so it got released as 0.4.0

**Pull request merged**